website/cue/reference/components/sinks/aws_kinesis_streams.cue

```diff
@@ -75,6 +75,7 @@ components: sinks: aws_kinesis_streams: components._aws & {
 
 	configuration: base.components.sinks.aws_kinesis_streams.configuration & {
 		_aws_include: false
+		request_retry_partial: warnings: ["This can cause duplicate logs to be published."]
```

Comment:
The original note was "Note: this can cause duplicates in Firehose." , just wanted to confirm - does this apply to both streams and firehose?

Reply:
Ah I think you're right - looking at https://github.com/vectordotdev/vector/pull/16771#issuecomment-1468467512

Reply:
Yes, stream should not need, removed it in stream.